Source/WPF/MyMoney/Views/TransactionsView.xaml.cs

```diff
@@ -1392,15 +1392,15 @@ public bool CheckTransfers()
             return true;
         }
 
-        private void ShowDanglingTransfers(List<Transaction> dangling)
+        private void ShowDanglingTransfers(HashSet<Transaction> dangling)
         {
             this.FireBeforeViewStateChanged();
             this.fixedList = null;
             this.lastQuery = null;
             this.SwitchLayout("TheGrid_TransactionFromDetails");
             this.SetActiveAccount(null, null, null, null, null);
             Account danglingAccount = new Account() { Type = AccountType.Cash };
-            IList data = new TransactionCollection(this.myMoney, danglingAccount, dangling, true, false, null);
+            IList data = new TransactionCollection(this.myMoney, danglingAccount, dangling.ToList<Transaction>(), true, false, null);

```

Comment:
since this is now in a hashset we should add a sort by date operation here so the transactions are displayed in a reasonable order.